src/components/listarTesis/ListarTesis.styles.js

```diff
@@ -0,0 +1,105 @@
+import { makeStyles } from "@material-ui/styles";
+import { CenterFocusStrong } from "@mui/icons-material";
+import { borderRadius } from "@mui/system";
+
+export const useStyles = makeStyles(() => ({
+    container: {
+        backgroundColor: "#fffff",
+        border: "1px solid #dfe0eb",
+        borderRadius: 3,
+
+    },
+    contentTable:{
+        width: '100%',
+        height:'100%',
+    },
+    tabletesis:{
+        width: '100%',
+        height:'100%',
+    },
+    titles: {
+        fontFamily: "Rubik",
+        fontStyle: "normal",
+        fontWeight: "600",
+        fontSize: 14,
+        color: "#000000",
```

Comment:
Nice now is good but you need to take the value of colors from the template 
`makeStyles((theme) => ({
    container: {
        backgroundColor: theme.palette.primary.main, 
        border: "1px solid #dfe0eb",
        borderRadius: 3,
`
this is only an example, you can add the colors that you need in to https://github.com/Ditmar/plagiarist-g1/blob/master/src/theme/theme.js file